tests/db/test_bigchain_api.py

```diff
@@ -1000,46 +887,22 @@ def test_multiple_owners_before_multiple_owners_after_single_input(self, b,
         tx_link = b.get_owned_ids(user_vk).pop()
         tx_input = b.get_transaction(tx_link.txid)
 
-        tx = Transaction.transfer(tx_input.to_inputs(), [[user3_vk, user4_vk]], tx_input.asset)
+        tx = Transaction.transfer(tx_input.to_inputs(),
+                                  [([user3_vk, user4_vk], 1)], tx_input.asset)
         tx = tx.sign([user_sk, user2_sk])
 
         assert b.is_valid_transaction(tx) == tx
         assert len(tx.fulfillments) == 1
         assert len(tx.conditions) == 1
 
-    @pytest.mark.skipif(reason=('Multiple inputs are only allowed for the '
```

Comment:
Yes, I handle all those cases in `test_divisible_assets.py`.
The only cases I didn't handle was the ones with `get_owned_ids` and `get_spent`